resources/jupyter_notebook_config.partial.py

```diff
@@ -1,12 +1,3 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-# Configuration file for ipython-notebook.
-
-c = get_config()
-c.NotebookApp.ip = '*'
-c.NotebookApp.open_browser = False
-c.NotebookApp.port = 8888
 
```

Comment:
This is the only part I'm questioning, but I'm assuming the originating files already have it and we're relying on it to stay consistent?